eva/catalog/models/base_model.py

```diff
@@ -108,7 +108,7 @@ def init_db():
         logger.info("Database does not exist, creating database.")
         create_database(engine.url)
         logger.info("Creating tables")
-        BaseModel.metadata.create_all()
+        BaseModel.metadata.create_all(checkfirst=True)
```

Comment:
What does this `checkfirst` option mean? Why is it now needed? We could add some comments here.

Reply:
Removed. It is true by default.